### PR TITLE
clean up logging

### PR DESCRIPTION
**Pull request by Ryang-21**

### CODE REVIEW

File: crates/artemis-core/src/executors/soroban_executor.rs [hunk 1]
```diff
@@ -167,39 +168,35 @@ async fn submit(
     });
 
     let res = rpc.send_transaction_polling(&signed_tx_envelope).await?;
-    info!("Submitted tx: {:?}\n", action.op.body.clone());
-    info!("Soroban response: {:?}", res.status);
+
     let log_msg = format!(
-        "Submitted tx: {:?} {:?} with response: {:?} {:?}\n",
+        "Submitted tx: {:?} with response: {:?}n",
```
Comment (mootz12) [inline crates/artemis-core/src/executors/soroban_executor.rs:173]: missing the new line here

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs [hunk 2]
```diff
@@ -248,8 +231,23 @@ impl BlendLiquidator {
...
+                    self.pending_fill.push(pending_fill.clone());
                 }
+
+                let status = if pending_fill.pct_to_fill > 0 {
+                    "picked up"
+                } else {
+                    "ignored"
+                };
+                let msg = format!(
+                    "Liquidator: {} has {} a new user liquidation auction for user: {:?} with lot: {:?} and bid: {:?}",
```
Comment (mootz12) [inline crates/strategies/blend-strategies/src/liquidation_strategy.rs:243]: we probably want to leave the `!<channel>` tag here for now, I think. It's important to get a phone notification when these get picked up still.

Comment (mootz12) [reply crates/strategies/blend-strategies/src/liquidation_strategy.rs:243]: (this log is a significant improvement tho)

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs [hunk 3]
```diff
@@ -334,10 +309,11 @@ impl BlendLiquidator {
                             .unwrap(),
                         )
                         .unwrap();
+
                     info!(" New pending bad debt fill: {:?}", pending_fill.clone());
-                    self.pending_fill.push(pending_fill);
+                    self.pending_fill.push(pending_fill.clone());
                     //we only care about lot here
-                } else {
+                } else if auction_type == 2 {
```
Comment (mootz12) [inline crates/strategies/blend-strategies/src/liquidation_strategy.rs:316]: Does this code path not handle user liquidations?

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs [hunk 4]
```diff
@@ -404,11 +402,8 @@ impl BlendLiquidator {
                     }
                     _ => Default::default(),
                 };
-                info!(
-                    "Auction filled, user: {:?}, liquidator {:?}",
-                    liquidated_id, liquidator_id
-                );
```
Comment (mootz12) [inline crates/strategies/blend-strategies/src/liquidation_strategy.rs:410]: This log still seems useful to have no?

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs [hunk 5]
```diff
@@ -363,9 +339,31 @@ impl BlendLiquidator {
...
+                    "picked up"
+                } else {
+                    "ignored"
+                };
+                let auction_type = if auction_type == 1 {
+                    "bad debt"
+                } else if auction_type == 2 {
+                    "interest"
+                } else {
+                    "unknown"
```
Comment (mootz12) [inline crates/strategies/blend-strategies/src/liquidation_strategy.rs:356]: ditto on the `user_liquidation` auctions

**Review by mootz12 — CHANGES_REQUESTED** [review]

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs (hunk 3 above)
Comment (Ryang-21) [reply crates/strategies/blend-strategies/src/liquidation_strategy.rs:316]: New auction events are only for bad debt and interest auctions. User liquidations come in their own event

File: crates/artemis-core/src/executors/soroban_executor.rs [hunk 6]
```diff
@@ -167,39 +168,35 @@ async fn submit(
...
+                action.signing_key.verifying_key().to_bytes(),
+            ))
+            .to_string(),
+            log_msg
             )
         })
         .to_string();
 
-        client
+        reqwest::Client::new()
```
Comment (mootz12) [inline crates/artemis-core/src/executors/soroban_executor.rs:194]: typo (occurs in multiple locations)

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs [hunk 7]
```diff
@@ -248,8 +231,20 @@ impl BlendLiquidator {
...
                 }
+
+                let msg = format!(
+                        "<!channel> - Liquidator: {} found a new user liquidation auction for user: {:?} with lot: {:?} bid: {:?} pct_to_fill: {:?} target_block: {:?}",
+                        self.us_public,
+                        user,
+                        auction_data.lot,
+                        auction_data.bid,
+                        pending_fill.pct_to_fill,
+                        pending_fill.target_block - pending_fill.auction_data.block
```
Comment (mootz12) [inline crates/strategies/blend-strategies/src/liquidation_strategy.rs:244]: will this underflow when target block is 0?

We should consider just logging both `auction_data.block` and `pending_fill.target_block`

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs (hunk 4 above)
Comment (mootz12) [reply crates/strategies/blend-strategies/src/liquidation_strategy.rs:410]: cc @Ryang-21

File: crates/strategies/blend-strategies/src/liquidation_strategy.rs [hunk 8]
```diff
@@ -362,10 +333,28 @@ impl BlendLiquidator {
...
+                    "unknown"
+                };
+                let msg = format!(
+                        "<!channel> - Liquidator: {} found a new {} auction with lot: {:?} bid: {:?} pct_to_fill: {:?} target_block: {:?}",
+                        self.us_public,
+                        auction_type,
+                        auction_data.lot,
+                        auction_data.bid,
+                        pending_fill.pct_to_fill,
+                        pending_fill.target_block - pending_fill.auction_data.block
```
Comment (mootz12) [inline crates/strategies/blend-strategies/src/liquidation_strategy.rs:354]: ditto on previous block comment

**Review by mootz12 — CHANGES_REQUESTED** [review]

**Review by mootz12 — APPROVED** [review]

**Pull request merged**